### DISCUSSION TITLE
How to use events with Stipple component?

### DISCUSSION THREAD

**Discussion by AbhimanyuAryan** [GenieFramework/Stipple.jl#95] [Q&A]
I'm specifically look at QSelect component https://v1.quasar.dev/vue-components/select#qselect-api @hhaensel 

Issue: 

Currently

```html
<q-select filled v-model="model" clearable use-input multiple counter use-chips label="Social Networks" :options="networks"></q-select>
```

appends a garbage value when typing input to select: 
<img width="587" alt="current" src="https://user-images.githubusercontent.com/8083613/157355053-acb139d5-acf3-4518-bd73-e026029b79d3.png">

What I trying to achieve is that once the user picks the option from drop. There's no garbage value appended(the one user typed)

This can be achieved by following code: 

```html
<q-select ref="mySelect" @update:model-value="mySelect.updateInputValue('')" filled v-model="model" clearable use-input multiple counter use-chips label="Social Networks" :options="networks"></q-select>
```

Codepen: https://codepen.io/milospaunovic/pen/BameGba?editors=1011

However I am unsure how to use @update event with `model-value`

This is also required for pkgviz to auto suggest package names: 

1) view: https://github.com/GenieFramework/PkgVizBoard/blob/8800ebf587cc786a61cbed28560be42b26ff7ffa/app/resources/dashboard/views/dashboard.jl#L13

2) js method: https://github.com/GenieFramework/PkgVizBoard/blob/8800ebf587cc786a61cbed28560be42b26ff7ffa/app/resources/dashboard/Dashboard.jl#L161-L175

**Comment by AbhimanyuAryan** [comment]
edit: vue2 1.15 UMD is using `@input -> function(value)` instead of `@update:model-value`

**Reply by AbhimanyuAryan** [reply]
```
<q-select ref="mySelect" @input="$refs.mySelect.updateInputValue('')" clearable use-input multiple counter use-chips label="Social Networks" :options="networks"></q-select>
```

**Comment by essenciary** [comment]
@AbhimanyuAryan Pretty cool! Seems like you found the solution? What's the question?

**Reply by AbhimanyuAryan** [reply]
@essenciary I have found the solution for vuejs. Now sure how to use quasar component's events in Stipple 

`@input -> function(value)`
